pkg/ingester/ingester.go

```diff
@@ -1505,8 +1502,9 @@ func (i *Ingester) UserStats(ctx context.Context, req *client.UserStatsRequest)
 	return createUserStats(db, req)
 }
 
-func (i *Ingester) AllUserStats(_ context.Context, req *client.UserStatsRequest) (*client.UsersStatsResponse, error) {
-	if err := i.checkRunning(); err != nil {
+func (i *Ingester) AllUserStats(_ context.Context, req *client.UserStatsRequest) (resp *client.UsersStatsResponse, auErr error) {
+	defer func() { auErr = i.handleReadError(auErr) }()
```

Comment:
Same comment as before, i.e. I think that the returned error should be named `err` to avoid confusion and potential programming mistakes.